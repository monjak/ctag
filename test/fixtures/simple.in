**********
1x Jace, the Mind Sculptor - 250 Lt
1x Searing Blaze - 9 Lt
1x Butcher of Malakir - 10 Lt
2x Claws of Valakut - 1 Lt
2x Crusher Zendikon - 1.5 Lt
1x Mordant Dragon - 8 Lt
1x Comet Storm (Worldwake Prerelease) - 8 Lt
2x Graypelt Hunter - 1 Lt
1x Pulse Tracker - 3 Lt
2x Join the Ranks - 3 Lt
