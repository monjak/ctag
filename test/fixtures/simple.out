**********
1x [c]Jace, the Mind Sculptor[/c] - 250 Lt
1x [c]Searing Blaze[/c] - 9 Lt
1x [c]Butcher of Malakir[/c] - 10 Lt
2x [c]Claws of Valakut[/c] - 1 Lt
2x [c]Crusher Zendikon[/c] - 1.5 Lt
1x [c]Mordant Dragon[/c] - 8 Lt
1x [c]Comet Storm[/c] (Worldwake Prerelease) - 8 Lt
2x [c]Graypelt Hunter[/c] - 1 Lt
1x [c]Pulse Tracker[/c] - 3 Lt
2x [c]Join the Ranks[/c] - 3 Lt
